### DISCUSSION TITLE
Can we do with D3.js something like this?

### DISCUSSION THREAD

**Discussion by sonsuzdejavu** [d3/d3#3494] [Q&A] [answered]
https://sanddance.js.org/app/

Millions of data points... 

Viewing data in 3d and move around data with righ click of mouse is also cool. (Click 3d cube top right.)

**Comment by Fil** [comment, marked as answer]
Yes. See for example https://blog.scottlogic.com/2020/05/01/rendering-one-million-points-with-d3.html
